docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -56,6 +56,10 @@ Property Name                                      Required   Description
                                                               default the location is ``/hbase``
 ``phoenix.config.resources``                       No         Comma-separated list of configuration files (e.g. ``hbase-site.xml``) to use for
                                                               connection properties.  These files must exist on the machines running Trino.
+``phoenix.max-scans-per-split``                    No         Maximum number of HBase scans allowed to be grouped into a single split. Default is 20.
+                                                              Lower values will lead to more splits in Trino.
+                                                              Can also be set via session propery ``phoenix.max_scans_per_split``.
```

Comment:
Remove the `phoenix` prefix since that'll depend on actual catalog name.

Reply:
Arrgh. Yes you are right of course.

Reply:
Should I rename this to "max_hbase_scans_per_split" or something? So that the general "max_scans_per_split" is available to other connectors if needed?

Reply:
I don't think that should be an issue. Session properties are already "namespaced". I think existing convention suggests to keep existing name.